cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -86,6 +92,21 @@ def _build_resource_name(self, name, to_lower=False):
             name = name.lower()
         return "-".join([name, self._stack_unique_id()])
 
+    def _build_image_recipe_name(self, to_lower=False):
+        name = "{0}-{1}".format(RESOURCE_NAME_PREFIX, self.image_name)[0:1024]
+        if to_lower:
+            name = name.lower()
+        return name
+
+    def _get_log_group_arn(self):
+        log_group_arn = self.format_arn(
+            service="logs",
+            resource="log-group",
+            region=utils.get_region(),
+            resource_name=":{0}/{1}/{2}".format("aws", "imagebuilder", self._build_image_recipe_name()),
```

Comment:
let's use f-strings when possible. 
Here you can simplify a lot the resource name `f":aws/imagebuilder/{self._build_image_recipe_name()}"`